prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -599,14 +599,36 @@ ssize_t efa_rdm_ep_trigger_handshake(struct efa_rdm_ep *ep, struct efa_rdm_peer
  */
 ssize_t efa_rdm_ep_post_handshake(struct efa_rdm_ep *ep, struct efa_rdm_peer *peer)
 {
+	struct efa_rdm_ope *txe;
+	struct fi_msg msg = {0};
 	struct efa_rdm_pke *pkt_entry;
 	fi_addr_t addr;
 	ssize_t ret;
 
 	addr = peer->efa_fiaddr;
+	msg.addr = addr;
+
+	/* ofi_op_write is ignored in handshake path */
+	txe = efa_rdm_ep_alloc_txe(ep, peer, &msg, ofi_op_write, 0,
```

Comment:
Does this extra txe allocation impact perf? Probably not... I understand you want to make all pkt type have a valid ope so you can save the peer lookup in the fast path.

Reply:
It makes the handshake path take slightly longer but it should amortize out, and our fast path should get faster.  I expect to see on the order of 2-3ns improvement from this PR in send path.